src/components/emailerFilterControls/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import moment from 'moment';
+import { EmailerFilters } from '../../containers/email/types';
+import {
+  Collapse,
+  Slider,
+  DatePicker,
+  AutoComplete,
+  Button,
+  message,
+} from 'antd';
+import { SliderMarks } from 'antd/lib/slider';
+import styled from 'styled-components';
+import { Neighborhoods } from '../../assets/content';
+import apiClient from '../../api/apiClient';
+import { CloseOutlined } from '@ant-design/icons';
+
+const StyledCollapse = styled(Collapse)`
+  max-width: 400px;
+`;
+
+const StyledRangePicker = styled(DatePicker.RangePicker)`
+  margin-right: 15px;
+`;
+
+const autoCompleteStyles = { width: 200, marginRight: 15 };
+
+interface EmailerFilterControlsProps {
+  filters: EmailerFilters;
+  setFilters: React.Dispatch<React.SetStateAction<EmailerFilters>>;
+}
+
+const MAX_COUNT = 10;
```

Comment:
❓Can we rename this to something like `MAX_ACTIVITY_COUNT` so that it's a bit more descriptive?